tests/testthat/test_formatting.R

```diff
@@ -19,3 +19,21 @@ test_that("Example numerical known answer", {
 test_that("Example character known answer", {
   expect_equal(format_ay("201112"), "2011/12")
 })
+
+context("round_nearest")
+
+test_that("Input as expected", {
+  expect_error(round_nearest(12, "ten"), "n must be a number")
+})
+
+test_that("Input as expected", {
+  expect_error(round_nearest("twelve", 10), "x must be a number")
+})
+
+test_that("Example numerical known answer", {
+  expect_equal(round_nearest(1337,5), "1335")
```

Comment:
Travis was playing up a bit but should be fixed now. These 2 latter tests fail as the output of your function is a double not a character. Can you change to 1335 and 1340